### DISCUSSION TITLE
FEATURE REQUEST: Add Search of NIST Control Family to Collection

### DISCUSSION THREAD

**Discussion by jeremywood-ai** [NUWCDIVNPT/stig-manager#519] [Ideas and Enhancements]
**Is your feature request related to a problem? Please describe.**

In an effort to consolidate similar controls, often times I will search on NIST Control Families, e.g., CM-2, to break the requirements for my teams. It is frustrating to scroll through just to find out there are 40 controls that are related.

**Describe the solution you'd like**
Add a search function for keyword or control family to allow for quicker tickets and better discussion with development teams.

**Describe alternatives you've considered**
An alterative would be free-form search that treat all text as searchable to find patterns, like CM-2.

**Additional context**
At the moment, there does not appear to be a way to filter out only specific control families. I have been able to filter on an "SV-" value and look around it for controls.

**Comment by cd-rite** [comment]
Hi @jeremywood-ai 

Could you be a little more specific about where you see this feature being implemented?  Would this be in the Asset Review workspace? Are you looking to filter checklists so you only see a view of Rules that are linked to a specific Control you search for?  

STIG Manager is pretty focused on STIGs and the Groups/Rules they contain.  We do provide some information regarding NIST Controls in the Rule Info box when you are looking at reviews, but they are sort of second-order members of our data set.  STIGs are composed of Rules, which link to CCI numbers, which THEN link to NIST Controls.   It sounds like you would want to come at it from the other direction: starting from a set of Controls, produce a set of STIGs or STIG rules that satisfy those controls.  This is a little out of STIGMans lane at the moment, but we will keep it in mind! Thanks!



Note: Converting to a discussion for any further....discussion on this feature request

**Reply by jeremywood-ai** [reply]
Hello, @cd-rite ,
I appreciate the conversation. While I am in the "Collection Review" area of specific STIG, I have CAT, Rule ID, Rule Title and other columns.

The way I intended this suggested feature would be allow for freeform search to allow rules to collect up into a NIST control families. At the moment, it is a constantly going over the list of rules to find a set of relates controls to take on.
